<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtKinectExample</class>
 <widget class="QWidget" name="QtKinectExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1621</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>view</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextEdit" name="outputText">
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>170</height>
      </size>
     </property>
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>LED</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="led_off">
           <property name="text">
            <string>OFF</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="led_green">
           <property name="text">
            <string>GREEN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="led_red">
           <property name="text">
            <string>RED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="led_yellow">
           <property name="text">
            <string>YELLOW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="led_blinkGreen">
           <property name="text">
            <string>BLINK_GREEN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="led_yellowRed">
           <property name="text">
            <string>BLINK_YELLOW_RED</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Tilt: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slide_Tilt">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="minimum">
             <number>-50</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="bt_clearPreview">
          <property name="text">
           <string>Clear preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showGamedata">
          <property name="text">
           <string>Game Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGraphicsView" name="previewScene">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1290</width>
       <height>500</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
